docs/requirement-sets/outlook/preview-requirement-set/outlook-requirement-set-preview.md

```diff
@@ -93,6 +93,32 @@ Added a new enum `AppointmentSensitivityType` that represents the sensitivity op
 
 ---
 
+### Additional message properties
+
+#### [Office.context.mailbox.item.inReplyTo](/javascript/api/outlook/office.messagecompose?view=outlook-js-preview&preserve-view=true#outlook-office-messagecompose-inreplyto-member)
+
+Added a property to get the message ID of the original message being replied to by the current message.
+
+**Available in**: Outlook on Windows (Microsoft 365 subscription)
+
+#### [Office.context.mailbox.item.getConversationIndexAsync](office.context.mailbox.item.md#methods)
+
+Added a method to get the Base64-encoded position of the current message in a conversation thread.
+
+**Available in**: Outlook on Windows (Microsoft 365 subscription)
+
+#### [Office.context.mailbox.item.getItemClassAsync](office.context.mailbox.item.md#methods)
```

Comment:
since this is only for message compose, should be able to link directly

```suggestion
#### [Office.context.mailbox.item.getItemClassAsync](/javascript/api/outlook/office.messagecompose?view=outlook-js-preview&preserve-view=true#outlook-office-messagecompose-getitemclassasync-member(1))
```